x-pack/plugin/ml/build.gradle

```diff
@@ -38,15 +38,18 @@ configurations {
   }
 }
 
-tasks.named("bundlePlugin").configure {
-  dependsOn configurations.nativeBundle
-  from {
-    configurations.nativeBundle.files.collect { zipTree(it) }
-  }
+esplugin.bundleSpec.from {
+  configurations.nativeBundle.files.collect { zipTree(it) }
+}
+
+// We don't ship the individual nativeBundle licenses - instead
+// they get combined into the top level NOTICES file we ship
+esplugin.bundleSpec.exclude 'platform/licenses/**'
 
-  // We don't ship the individual nativeBundle licenses - instead
-  // they get combined into the top level NOTICES file we ship
-  exclude 'platform/licenses/**'
+["bundlePlugin", "explodedBundlePlugin"].each { bundleTaskName ->
```

Comment:
Ug, I hate that Gradle doesn't have solution to the use case of unzipping a configuration of multiple archives. It's insane at this point.